src/commands/utils/uploadArtifactOrUDF.ts

```diff
@@ -343,3 +343,125 @@ export async function executeCreateFunction(
   const createdMsg = `${userInput.functionName} function created successfully.`;
   void showInfoNotificationWithButtons(createdMsg);
 }
+interface QuickPickWizardItem extends vscode.QuickPickItem {
+  value: string;
+}
```

Comment:
Can we use [`QuickPickItemWithValue`](https://github.com/confluentinc/vscode/blob/main/src/quickpicks/types.ts) instead?